pkg/systemd/services/service-details.jsx

```diff
@@ -536,7 +530,25 @@
 
         const tooltipMessage = enabled ? _("Stop and disable") : _("Start and enable");
         const hasLoadError = this.props.unit.LoadState !== "loaded" && this.props.unit.LoadState !== "masked";
-        const loadError = this.props.unit.LoadError ? this.props.unit.LoadError[1] : null;
+
+        if (hasLoadError) {
+            const path = "/system/services" + (this.props.owner === "user" ? "#/?owner=user" : "");
+            const loadError = this.props.unit.LoadError ? this.props.unit.LoadError[1] : null;
+            const title = loadError || _("Failed to load unit");
+
+            return <EmptyStatePanel
+                icon={ExclamationCircleIcon}
+                title={title}
+                paragraph={this.props.unitId}
+                action={
+                    <Button variant="link"
+                            component="a"
+                            onClick={() => cockpit.jump(path, cockpit.transport.host)}>
+                        {_("View all services")}
+                    </Button>
+                }
+            />;
+        }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18416-20230227-161258-e3110a7d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)